src/rpc/Errors.cpp

```diff
@@ -77,7 +78,8 @@ getErrorInfo(ClioError code)
         {ClioError::rpcMALFORMED_REQUEST, "malformedRequest", "Malformed request."},
         {ClioError::rpcMALFORMED_OWNER, "malformedOwner", "Malformed owner."},
         {ClioError::rpcMALFORMED_ADDRESS, "malformedAddress", "Malformed address."},
-        {ClioError::rpcINVALID_HOT_WALLET, "invalidHotWallet", "Invalid hot wallet."}};
+        {ClioError::rpcINVALID_HOT_WALLET, "invalidHotWallet", "Invalid hot wallet."},
+        {ClioError::rpcINVALID_API_VERSION, JS(invalid_API_version), "Invalid API version."}};
```

Comment:
I'd like to use `invalidAPIVersion` or `invalidVersion` instead but this is what `rippled` returns so it may have to stay this way.